recipes/pocketfft/all/conandata.yml

```diff
@@ -0,0 +1,9 @@
+sources:
+  "cci.20240710":
```

Comment:
Same thing about the new version scheme for new recipes. If they have not released any version, we can go for `0.0.0.cci.20240710`

Reply:
Ok, sounds good.
Just to clarify - the version prefix should be the latest officially released version in general, right?

Reply:
Yes, so that if a new version is ever released, there's little chance of ours being considered newer than the released tag :) (And 0.0.0 for libraries without versions!)